Core/DISET/private/Transports/SSL/SocketInfo.py

```diff
@@ -65,8 +65,8 @@ def gatherPeerCredentials( self ):
         identitySubject = peerChain.getIssuerCert()['Value'].getSubjectNameObject()[ 'Value' ]
     else:
       identitySubject = peerChain.getCertInChain( 0 )['Value'].getSubjectNameObject()[ 'Value' ]
-    credDict = { 'DN' : identitySubject.one_line(),
-                 'CN' : identitySubject.commonName,
+    credDict = { 'DN' : str(identitySubject),
+                 'CN' : str(identitySubject).split('CN')[1][1:],
```

Comment:
you can do only one conversion at line 67. But is needed after all?

Reply:
Changed to one conversion. It is needed (at least for now) because other parts of code expect it to be a string, not a X509Name object. But this is a good time to redesign and use string only when printing, etc. Added to my to-do list.